src/js/components/stock-movement-wizard/modals/AdjustInventoryModal.jsx

```diff
@@ -90,12 +90,8 @@ class AdjustInventoryModal extends Component {
     this.props.showSpinner();
 
     const url = '/openboxes/api/stockAdjustments';
-
-    return apiClient.post(url, _.map(values.adjustInventory, (adItem) => {
-      const adjustItem = _.find(
-        _.filter(this.state.attr.fieldValue.adItem, listItem => !listItem.initial),
-        item => item['inventoryItem.id'] === adItem['inventoryItem.id'],
-      );
+    const payload = _.map(values.adjustInventory, (adItem) => {
+      const adjustItem = _.find(item => item['inventoryItem.id'] === adItem['inventoryItem.id']);
```

Comment:
I think that you need to specify a collection to inspect (https://lodash.com/docs/4.17.10#find)